Panto/css/style.css

```diff
@@ -50,18 +54,97 @@
 .hero {
   text-align: center;
   color: var(--color-white);
+  padding-block: 16rem;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  gap: 4.4rem;
+}
+
+.hero-text-box {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  gap: 2.4rem;
+}
+
+.hero-text-box h1 {
+  width: 80%;
+}
+
+.hero-description {
+  width: 45%;
+  font-size: var(--font-size-hero-description);
+  line-height: var(--line-height-hero-footer-p);
+  opacity: 0.8;
+}
+
+.search-bar {
+  position: relative;
+  backdrop-filter: blur(0.3rem);
+  border-radius: 4.2rem;
+  width: 32rem;
 }
 
 .search-bar input[type="search"] {
   font-family: inherit;
+  font-size: var(--font-size-search-bar);
+  background-color: var(--color-search-bar-background);
+  opacity: 0.8;
+  padding: 1.6rem 12.8rem 1.6rem 1.6rem;
```

Comment:
what's with the 12.8 padding on the right side?
![image](https://github.com/user-attachments/assets/019ac400-2ab2-41af-8af4-fdaf429418e6)
it makes your input look like this

and you have also forgotten to change the `color` for input text